spec.html

```diff
@@ -23369,7 +23369,7 @@ <h1>isNaN ( _number_ )</h1>
         1. Otherwise, return *false*.
       </emu-alg>
       <emu-note>
-        <p>A reliable way for ECMAScript code to test if a value `X` is a *NaN* is an expression of the form `X !== X`. The result will be *true* if and only if `X` is a *NaN*.</p>
+        <p>A reliable way for ECMAScript code to test if a value *X* is a *NaN* is an expression of the form `X !== X`. The result will be *true* if and only if *X* is a *NaN*.</p>
```

Comment:
There isn't a spec value named `X`. The sentence refers to it as a value, but then uses it to construct an expression, so it seems more appropriate to treat it as code (i.e., keep the backticks).

We could also change the wording slightly:
```
    ... test if the value of `X` ...
```